schema/2.0.0/corpora_schema.md

```diff
@@ -905,6 +906,7 @@ schema v2.0.0 substantially *remodeled* schema v1.1.0:
 * var
   * Replaced HGNC gene **symbols** as `var.index` with ENSEMBL or ERCC spike-in **identifiers** 
   * Added `feature_id`, `feature_name`, and `feature_reference`
+  * Added requirements for `raw.var` which should be identical to `var`
```

Comment:
To be precise, should->must

Reply:
Is this still in discussion? Does var index need to be _identical_ to raw.var index or can it be a _subset of_ raw.var index?